quickwit/quickwit-janitor/src/actors/delete_task_planner.rs

```diff
@@ -35,9 +35,13 @@ use quickwit_metastore::{
 use quickwit_proto::metastore_api::DeleteTask;
 use quickwit_proto::SearchRequest;
 use quickwit_search::{jobs_to_leaf_request, SearchClientPool, SearchJob};
+use serde::Serialize;
+use tantivy::Inventory;
 use tracing::{debug, info};
 
-const PLANNER_REFRESH_INTERVAL: Duration = Duration::from_secs(60 * 60);
+use crate::metrics::JANITOR_METRICS;
+
+const PLANNER_REFRESH_INTERVAL: Duration = Duration::from_secs(60);
```

Comment:
This is voluntary I assume?

Reply:
Yes, now we can refresh every minute basically, we won't resend several times the delete operations.